zerver/actions/message_delete.py

```diff
@@ -44,7 +44,7 @@ def check_update_first_message_id(
     send_event_on_commit(realm, stream_event, users_to_notify)
 
 
-def do_delete_messages(realm: Realm, messages: Iterable[Message]) -> None:
+def do_delete_messages(realm: Realm, messages: Iterable[Message], user_profile: Optional[UserProfile] = None) -> None:
```

Comment:
Should this be named something that tells us that this is the user deleting the messsage?

Reply:
Changes made